### PR TITLE
Solution for hm23

### PR DESCRIPTION
**Pull request by MaksymMatuliak**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -11,6 +11,17 @@
  * - В классе Main создать добавить по одному екземпляру каждого класса.
  * - Отправить все машини на работу и остановить работу всех машин.
  */
-public class MainApp {
 
+public class MainApp {
+    public static void main(String[] args) {
+        Track t = new Track();
+        Bulldozer b = new Bulldozer();
+        Excavator e = new Excavator();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MainApp.java]:
```suggestion
        Machine t = new Track();
        Machine b = new Bulldozer();
        Machine e = new Excavator();
```
Also, t,b,e - not informative naming

Comment (MaksymMatuliak) [reply src/main/java/core/basesyntax/MainApp.java]: done

File: src/main/java/core/basesyntax/Track.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Track extends Machine {
+    @Override
+    void doWork() {
+        System.out.println("Buldozer is working...");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Track.java]: копіпаста - зло :)

Comment (MaksymMatuliak) [reply src/main/java/core/basesyntax/Track.java]: done)

**Pull request closed without merging**